packages/devextreme/testing/tests/DevExpress.ui.widgets/loadPanel.tests.js

```diff
@@ -40,56 +41,193 @@ const LOADPANEL_MESSAGE_CLASS = 'dx-loadpanel-message';
+                loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
+                assert.strictEqual(loadPanelMessage, '', 'Load Panel message value is empty');
+            });
+        });
+
+
+        QUnit.test('message is set to a sample value on init', function(assert) {
+            const loadPanel = $('#loadPanel').dxLoadPanel({
+                visible: true,
+                message: 'Test',
+                showIndicator: false
+            }).dxLoadPanel('instance');
+            const loadPanelMessage = loadPanel.$wrapper().find('.dx-loadpanel-message').text();
+
+            assert.strictEqual(loadPanelMessage, 'Test', 'Load Panel message value is non-empty and correct value');
+        });
+
+        QUnit.test('messageLoacalization should display correct value', function(assert) {
+            const locale = localization.locale();
+            messageLocalization.load({
+                'en': {
+                    'Loading': 'Test'
+                }
+            });
+            localization.locale('en');
+            const loadPanel = $('#loadPanel').dxLoadPanel({
+                visible: true,
+                showIndicator: false
+            }).dxLoadPanel('instance');
+            const loadPanelMessage = loadPanel.$wrapper().find(`.${LOADPANEL_MESSAGE_CLASS}`).text();
+
+            assert.strictEqual(loadPanelMessage, messageLocalization.format('Loading'), 'message localization is showing correct value on initialization');
+
+            localization.locale(locale);
+        });
     });
 
-    QUnit.test('role on wrapper', function(assert) {
-        const instance = $('#loadPanel').dxLoadPanel({ visible: true }).dxLoadPanel('instance');
+    QUnit.module('wrapper / load indicator aria-label tests', () => {
+        QUnit.test('wrapper aria label is removed and insert into load indicator when showIndicator = true on runtime', function(assert) {
+            const loadPanel = $('#loadPanel').dxLoadPanel({
+                visible: true,
+                showIndicator: false
+            }).dxLoadPanel('instance');
+            const $wrapper = loadPanel.$wrapper();
+            let $indicator = loadPanel.$content().find(LOADINDICATOR_CLASS);
+            let indicatorAriaLabelExists = $indicator.is('[aria-label]');
+            let wrapperAriaLabelExists = $wrapper.is('[aria-label]');
+
+            assert.strictEqual(wrapperAriaLabelExists, true, 'Wrapper aria label is added');
+            assert.strictEqual(indicatorAriaLabelExists, false, 'Indicator aria label is empty');
+
+            assert.timeout(1000);
+            loadPanel.option('showIndicator', true);
+            $indicator = loadPanel.$content().find(LOADINDICATOR_CLASS);
+            indicatorAriaLabelExists = $indicator.is('[aria-label]');
+            wrapperAriaLabelExists = $wrapper.is('[aria-label]');
+
+            assert.strictEqual(wrapperAriaLabelExists, false, 'Wrapper aria label is removed after runtime');
```

Comment:
We can write $indicator.attr('aria-label') directly

Reply:
[bb57796](https://github.com/DevExpress/DevExtreme/pull/27080/commits/bb57796b0f4e0953f491b0adee68e1f35762a715)